.github/workflows/test-ruby.yml

```diff
@@ -184,6 +184,8 @@ jobs:
       DISABLE_SIMPLECOV: true
       RAILS_ENV: test
       BUNDLE_WITH: test
+      LOCAL_DOMAIN: localhost:3000
```

Comment:
One future change we should make would be to align these values for the regular spec suite and the fullstack system specs, so that we don't need to set them here. Ideally they'd be read from `.env.test` or something, and use same values across the whole suite.

Reply:
I think that would make sense, but I'm also a bit wary of the extensive changes in the test suite this may require.

Reply:
Yeah ... there are ~50 references to that `ngrok.io` host value across spec dirs. I suspect that some of that are using that just because it happens to be the value, not because the spec cares about the hostname. Maybe I'll do a quick spike branch on changing that to align the values and see how it goes.

Reply:
Not terrible - https://github.com/mastodon/mastodon/compare/main...mjankowski:align-hostnames - this WIP branch has a few edge case failures where I'm not sure on best approach to resolve, but the diff here is not that bad. I'm going to call this resolved for purposes of this PR, but may nudge this forward separately.